tests/gitbook.test.ts

```diff
@@ -346,3 +259,17 @@ test('GET spaces/space_iphone-doc/revisions/somerevision/files?limit=-1 (invalid
     });
     expect(result instanceof ValidationError ? result.path : null).toEqual(['query', 'limit']);
 });
+
+test('GET orgs/xxx/synced-blocks?ids=foo parsed as array', () => {
+    const result = validateRequest({
+        path: '/orgs/xxx/synced-blocks',
+        method: 'get',
+        headers: {
+            'content-type': 'application/json',
+        },
+        query: {
+            ids: 'foo',
+        },
+    });
+    expect(result.query).toEqual({ ids: ['foo'] });
+});
```

Comment:
Should you also add a test where you pass `ids: ['foo']` and one with `ids: ['foo', 'bar']`?

Reply:
Yeah 👍